node_modules/oae-core/members/js/members.js

```diff
@@ -46,7 +46,7 @@ define(['jquery', 'oae.core', 'jquery.history'], function($, oae) {
 
             var url = '/api/group/' + widgetData.context.id + '/members';
             if (query) {
-                url = '/api/search/members/'  + widgetData.context.id;
+                url = '/api/search/group-members-library/'  + widgetData.context.id;
```

Comment:
Not sure of the context, but are we changing the API endpoint from `/api/search/members/g:xx:xxx` to `/api/search/members-library/g:xx:xxx` or are we just adding a new endpoint?

If the former, that's a problem. The STEM incubator site relies on the existing API endpoint. If that endpoint goes away, then the site breaks.

If this is just a copy-and-paste issue and the existing API endpoint is sticking around, then never mind.